client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -35,8 +35,11 @@ export default function IssueMultipleLicensesForm( {
 	// after the payment method is added, we will make an attempt to issue the chosen license automatically.
 	const defaultProducts = getQueryArg( window.location.href, 'products' )?.toString().split( ',' );
 
-	const [ selectedProducts, setSelectedProducts ] = useState( defaultProducts ?? [] );
-	const [ issueLicenses, isLoading ] = useIssueMultipleLicenses( selectedProducts, selectedSite );
+	const [ selectedProductSlugs, setSelectedProductSlugs ] = useState( defaultProducts ?? [] );
```

Comment:
I renamed "selectedProducts" to "selectedProductSlugs" in order to make clear the difference between product slugs and product objects.

Reply:
And the same thing for "defaultProducts" to "defaultProductSlugs"